components/navigation/footer.module.css

```diff
@@ -28,6 +28,8 @@
 
 .SocialNetworks {
   @apply flex items-center gap-4;
+  flex-flow: row wrap;
+  justify-content: center;
```

Comment:
@MathCatsAnd I'd use Tailwind utility classes here, to keep it consistent with the rest of our codebase. So, instead of:

```css
.SocialNetworks {
  @apply flex items-center gap-4;
  flex-flow: row wrap;
  justify-content: center;
}
```

We could do:

```css
.SocialNetworks {
  @apply flex flex-wrap justify-center items-center gap-4;
}
```

And it should give us the same result 🙂